script/setup

```diff
@@ -17,15 +17,15 @@ fi
 mkdir -p config
 
 if [ ! -n "$DEVCONTAINER" ] && [ ! -n "$VIRTUAL_ENV" ];then
-  python3 -m venv venv
+  python3.11 -m venv venv
```

Comment:
Change back

Reply:
undid the changes to scripts/setup: 59e9c6c5e1dca46513101088e55753e00c73ced9